### PR TITLE
add ga note

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: docs/faq.md [hunk 1]
```diff
@@ -42,3 +42,11 @@ You can see graduated experiments by clicking on the `View Past Experiments` but
 Test Pilot bugs can be filed on the Test Pilot GitHub repo. [link](https://github.com/mozilla/testpilot/issues)
 
 You will find a link to report bugs for individual experiments on the experiment's detail page on [Test Pilot](https://testpilot.firefox.com/) for the most up to date link.
+
+## Why do Test Pilot experiments ask my to allow Google Analytics?
```
Comment (ghost) [inline docs/faq.md]: s/my/me/

File: docs/faq.md [hunk 2]
```diff
@@ -42,3 +42,11 @@ You can see graduated experiments by clicking on the `View Past Experiments` but
...
+
+Test Pilot uses GA event tracking to measure certain interactions within experiments. We use this data to better understand how participants are using experiments. Each experiment contains a summary description of what data we collect and a link to the precise schema definitions we send to GA. For example, here is the scheme for [Notes](https://github.com/mozilla/notes/blob/master/docs/metrics.md). We do not provide any Test Pilot data for use in Google's products & services.
```
Comment (ghost) [inline docs/faq.md]: s/scheme/schema/

**Review by ghost — APPROVED** [review]
r+ w/ changes

File: docs/faq.md [hunk 3]
```diff
@@ -42,3 +42,11 @@ You can see graduated experiments by clicking on the `View Past Experiments` but
...
+
+If you're uncomfortable with submitting data to GA, all of our experiments respect the DNT flag in `about:preferences`.
```
Comment (chuckharmston) [inline docs/faq.md]: "Do Not Track" instead?

Comment (johngruen) [reply docs/faq.md]: Good call

**Pull request merged**